odh/base/monitoring/github-receiver-service.yaml

```diff
@@ -0,0 +1,17 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: github-receiver-service
+  namespace: default
```

Comment:
can we remove this namespace -- we overwrite it anyway in the root kustomization